x/campaign/keeper/msg_server_add_mainnet_vesting_account_test.go

```diff
@@ -0,0 +1,201 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/campaign/types"
+	profiletypes "github.com/tendermint/spn/x/profile/types"
+)
+
+func TestMsgAddMainnetVestingAccount(t *testing.T) {
+	var (
+		addr1 = sample.AccAddress()
+		addr2 = sample.AccAddress()
+
+		coordAddr1 = sample.AccAddress()
+		coordAddr2 = sample.AccAddress()
+		coordAddr3 = sample.AccAddress()
+
+		campaign1 = sample.Campaign(0)
+		campaign2 = sample.Campaign(1)
+		campaign3 = sample.Campaign(2)
+
+		campaignKeeper, _, campaignSrv, profileSrv, sdkCtx = setupMsgServer(t)
+		ctx                                                = sdk.WrapSDKContext(sdkCtx)
+	)
+
+	// create shares
+	allocatedShares, err := types.NewShares("999token")
+	require.NoError(t, err)
+	totalShares, err := types.NewShares("9999token")
+	require.NoError(t, err)
+	share1, err := types.NewShares("9999token")
```

Comment:
Same general comment as ones in https://github.com/tendermint/spn/pull/304. We should use more explicit variable names for tests readability